tests/unit/test_service.py

```diff
@@ -614,6 +642,13 @@ def get_course_creators_by_uni(self, university):
                 result.append(i)
         return result
 
+    def get_course_start_by_course(self, course_id) -> DM.CourseStart:
```

Comment:
Expression of type "list[Unknown]" is incompatible with return type "CourseStart"
  "list[Unknown]" is incompatible with "CourseStart"PylancereportReturnType

Reply:
Liegt auch wieder mit dem problem zuvor zusammen course_start.course_id kann nicht zugegriffen werden...